### ISSUE TITLE
mp4 meta tags not visible after transcoding with ffmpeg, but ffprobe shows them

### ISSUE THREAD

**Issue by dzek69** [MediaArea/MediaInfo#453]
I've transcoded a movie file with ffmpeg, keeping meta tags, and I can see those tags when using ffprobe on output file. Using MediaInfo I can see those tags with input file but not with output file.

Input file Media Info:
https://pastebin.com/3fwrQa1D
Output file Media Info: 
https://pastebin.com/kydTRDim

So it still reads `Encoded date` in General section, but cannot read `com.android.version` or location.

`ffprobe` on the input file:
https://pastebin.com/tPPE7TWe
`ffprobe` on the output file:
https://pastebin.com/CgMWQ0P2

Output file:
[VID_20200404_151129.encoded.zip](https://github.com/MediaArea/MediaInfo/files/4691072/VID_20200404_151129.encoded.zip)

Transcoding command used:
```
ffmpeg -n -i ../../Pictures/Photos/__Unsorted/Xiaomi - lepszy/2020/04/VID_20200404_151129.mp4 -c:v libx264 -preset slow -crf 23 -c:a aac -ss 5 -to 7 -vsync 2 -tune film -movflags +faststart -map_metadata 0 -movflags use_metadata_tags VID_20200404_151129.encoded.mp4
```

**Comment by JeromeMartinez** [comment]
When adding the code corresponding to the parsing of the missing "keys", I find more metadata than in your input file:

~~~
major_brand                              : mp42
minor_version                            : 0
compatible_brands                        : isommp42
creation_time                            : 2020-04-04T13:11:42.000000Z
location                                 : +49.9253+022.0787/
location-eng                             : +49.9253+022.0787/
com.android.version                      : 9
encoder                                  : Lavf58.20.100
~~~

Would you mind to share the input file too so I can compare an add more data too (I don't like the "xyz: [moderated]" line in your export)?

**Comment by dzek69** [comment]
i just modarated recording location for privacy reasons I totally forgot that they are readable from the output file I've attached 🤦... looks like I'm not that good at privacy

At least let me keep the original recording for myself and have this another file instead - both original and converted are included in this zip:
https://dl.dropboxusercontent.com/s/roz2qvoft1jitx5/fire.zip

The same problem occurs on this file (the same thing recorded it - Xiaomi Redmi Note 5)

If there is some extra data in the output file than input file then ffmpeg probably added something, but I don't mind.

**Comment by JeromeMartinez** [comment]
> i just modarated recording location for privacy reasons

Oops... I deleted the attached file as well as my report, people monitoring this tracker still have the information but at least other people and robots will not have it.

> If there is some extra data in the output file than input file then ffmpeg probably added something, but I don't mind.

For a reason I ignore, looks like FFmpeg adds the major_brand of the source file in the output file metadata, looks like a bad side effect of your command line options. why not... Not sure what to do with that, maybe just trash them as it is not so useful, or at least flag them as source file info.

Your command line makes me see that FFmpeg is not a friend of Elon Musk, FFmpeg developers expect that [all videos are shot on Earth](https://github.com/FFmpeg/FFmpeg/blob/638b5d51cf65b384e8632e650056ed0e62b16884/libavformat/movenc.c#L3518) :)

**Comment by dzek69** [comment]
> I deleted the attached file as well as my report (...)

I did not know that you could remove your edit from the history. Nice, thanks GitHub. I thought it will be there forever so another file I've posted has this location included too. So I've removed this as well, I hope you've stored this on your computer, if not - I can record something for you without location and post it.

> looks like FFmpeg adds the major_brand (...)

I really have no idea what that meta tag is for :) Actually for my personal use I just care about location to get copied when I'm transcoding the files. If some other meta data gets copied - cool, maybe it will be useful in the future.

> looks like a bad side effect of your command line options

After a quick research I see people complaining about that, could be ffmpeg problem [?] https://stackoverflow.com/questions/17886516/ffmpeg-editing-metadata-major-brand

> FFmpeg developers expect that all videos are shot on Earth :)

Well, I hope it will get fixed before my trip to Mars, I don't want bad meta data in my files :D

**Comment by JeromeMartinez** [comment]
> After a quick research I see people complaining about that, could be ffmpeg problem [?] 

No, the poster didn't understand that FFmpeg is a remuxer and not a in place editor.
In the output FFmpeg stored the source major_brand value (a more technical field) in a metadata for whatever reason. Anyway not important, I see what is the issue about the location in the output file.

**Comment by dzek69** [comment]
Don't forget the `com.android.version` as well :) Thanks

**Comment by cjee21** [comment]
This is fixed already in current MediaInfo version. Using the exact FFmpeg command as in first post, the resulting video has Android version and GPS coordinates when viewed in MediaInfo.

**Comment by JeromeMartinez** [comment]
Seems OK now, closing.
